File: api/src/main/java/api/security/model/LoginRequest.java
package api.security.model;

import javax.validation.constraints.NotBlank;

public class LoginRequest {

    @NotBlank(message = "Username is absent")
    private String username;
    @NotBlank(message = "Password is absent")
    private String password;

    public String getUsername() {
        return username;
    }

    public LoginRequest setUsername(String username) {
        this.username = username;
        return this;
    }

    public String getPassword() {
        return password;
    }

    public LoginRequest setPassword(String password) {
        this.password = password;
        return this;
    }
}

File: api/src/main/java/api/security/model/LoginRequest.kt
package api.security.model

import javax.validation.constraints.NotBlank
import javax.validation.constraints.Size

open class LoginRequest {

    @NotBlank(message = "Username is absent")
    @Size(max = 40, min = 4, message = "Length must be between 4 and 40")
    var username: String = ""

    @NotBlank(message = "Password is absent")
    @Size(max = 40, min = 6, message = "Length must be between 6 and 40")
    var password: String = ""
}
